db/views/return_articles_v01.sql

```diff
@@ -0,0 +1,13 @@
+Select Articles.id as article_id, 
+	Users.id as user_id, 
+	Users.email, 
+	Tags.name, 
+	Articles.status, 
+	Articles.language, 
+	COUNT(ck_editor_images.file ) 
+From Articles
+INNER Join article_tags ON Articles.id = article_tags.id
```

Comment:
Why` inner join`? Why not` left join`?

Reply:
left join returns all rows from the left table. it returns even if there are no matches with the right table
inner join returns all rows from both tables, which have matches
So, I decided to use inner join, because we need to return articles with tags

Reply:
oh, my fault
also, I forgot to show all tags 

fixed